### ISSUE TITLE
AMR website is down

### ISSUE THREAD

**Issue by nschneid** [amrisi/amr-guidelines#276]
https://amr.isi.edu/ - who maintains this currently?

**Comment by cbonial** [comment]
It had been moved to a link off of Ulf’s ISI website, but now I can’t find it there either. Perhaps we need to consider if there is support to host it elsewhere? I still use the editor regularly and it’s extremely fast and convenient, if not critical, for expanding resources. There is also the CU tool, CAMRA, for editing AMRs, but it lacks shortcuts and I think checking heuristics of ISI editor.Best,ClaireSent from my iPhoneOn Aug 13, 2024, at 1:57 PM, Nathan Schneider ***@***.***> wrote:﻿
https://amr.isi.edu/ - who maintains this currently?

**Comment by flipz357** [comment]
Anyone got an update for this? If there's no quick solution, I'd propose all related links be taken out from the guidelines for now, to not risk giving off an unfortunate look.

Maybe it's also a good time to clarify the license situations of some AMR-related files, like the [framedescriptions](https://amr.isi.edu/doc/propbank-amr-frames-arg-descr.txt). I remember discussion, where basically nobody knew what's the license situation. E.g., if the license for a file like this is public, it could be directly hosted in this github here, which would be pretty nice. The more "native" AMR resources (frame descriptions, annotator tool, maybe little prince and bio amr) can be hosted in this github (or linked from this repo to other github repos), the better I'd say.

**Comment by jheinecke** [comment]
> It had been moved to a link off of Ulf’s ISI website, but now I can’t find it there either. Perhaps we need to consider if there is support to host it elsewhere? I still use the editor regularly and it’s extremely fast and convenient, if not critical, for expanding resources. There is also the CU tool, CAMRA, for editing AMRs, but it lacks shortcuts and I think checking heuristics of ISI editor.Best,ClaireSent from my iPhoneOn Aug 13, 2024, at 1:57 PM, Nathan Schneider ***@***.***> wrote:﻿ https://amr.isi.edu/ - who maintains this currently? —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Some stuff produced by Ulf is on https://uhermjakob.github.io/amr/. Ideally all should be in this repo (when licences allow)

**Comment by flipz357** [comment]
Any update on this? Seems like the website is still down. What's about the chance that it will be soon up again?

**Comment by jheinecke** [comment]
I heard it is permanently down. We need a new entry point (yours (https://github.com/flipz357/AMR-World) looks to be a great candidate !

**Comment by cbonial** [comment]
Has anyone reached out to Ulf? I’d be happy to do so. Do we have a proposal to ask about? I’m unsure as to whether there might be intellectual property issues, but it would be a great service to the community to have the editor up somewhere.Best,ClaireSent from my iPhoneOn Aug 30, 2024, at 5:27 AM, Johannes Heinecke ***@***.***> wrote:﻿
I heard it is permanently down. We need a new entry point (yours (https://github.com/flipz357/AMR-World) looks to be a gread candidate !

**Comment by flipz357** [comment]
Great suggestion, @cbonial, can you ask Ulf? I also don't have a concrete proposal, but it'd be great to have a common github to host files like the LP corpus, the original editor, and include links to other useful AMR github tools, e.g., Johannes's [new AMR editor](https://github.com/Orange-OpenSource/metamorphosed)! I also have some of these files somewhere, so if the license situation is clear, I can also bring them online, e.g., in the mentioned [AMR world](https://github.com/flipz357/AMR-World) repo.

**Comment by jheinecke** [comment]
As mentioned above, useful document of AMR roles can be found on Ulf's Website (https://uhermjakob.github.io/amr/lib/roles.html) Shouldn't we integrate this here for roles which are undocumented sofar in amr.md (if you think this is useful? I can do the maodifcations and do pull request)

**Comment by cbonial** [comment]
Yes, I think that’s would be useful, thanks Johannes! I have emailed Ulf to see if we can discuss hosting the editor somewhere else or what the options may be. I’ll keep you posted!ClaireSent from my iPhoneOn Sep 13, 2024, at 3:11 AM, Johannes Heinecke ***@***.***> wrote:﻿
As mentioned above, useful document of AMR roles can be found on Ulf's Website (https://uhermjakob.github.io/amr/lib/roles.html) Shouldn't we integrate this here for roles which are undocumented sofar in amr.md (if you think this is useful? I can do the maodifcations and do pull request)

**Comment by flipz357** [comment]
@cbonial Have you received an answer from Ulf?

I also see that @austinjp has repaired some link with a PR, linking to Ulf's personal HP. This is good, but it seems to me that his personal HP may not be a full replacement for the AMR webpages https://amr.isi.edu/, which contained/linked to much more resources.

**Comment by cbonial** [comment]
Hi Juri, No, I haven’t received a reply. I will ping again on this; thanks, this helped to remind me it had been some time! ClaireSent from my iPhoneOn Oct 25, 2024, at 6:27 AM, Juri Opitz ***@***.***> wrote:﻿
@cbonial Have you received an answer from Ulf?
I also see that @austinjp has repaired some link with a PR, linking to Ulf's personal HP. This is good, but it seems to me that his personal HP may not be a full replacement for the AMR webpages https://amr.isi.edu/, which contained/linked to much more resources.

**Comment by flipz357** [comment]
I have these files on my computer and could post them in a repository and then propose placing a link to them here.

_Little Prince:_ amr-bank-struct-v3.0.txt
_Bio AMR:_ amr-release-bio-v3.0.txt
_Propbank frame description as txt file:_ propbank-amr-frames-arg-descr.txt

They were publicly downloadable from https://amr.isi.edu/. The last file was also linked in amr.md (that linked to the broken webpage, so this link stopped working). Do you think it is okay to upload these files? I am not a 100% clear on the license, but these documents used to be public before the webpage broke...

Some other links are fixed by @austinjp in a #278 , who links to Ulf's personal page. But afaik the above files are not on his personal HP.

**Comment by nschneid** [comment]
The Wayback Machine has them.

**Comment by flipz357** [comment]
Good spot. Are you saying that the broken links should be simply redirected to the WayBack machine?

**Comment by nschneid** [comment]
Up to you, I was just pointing out where the data can be found.

**Comment by flipz357** [comment]
Great, I think there's now two pull requests that fix the broken links.

**Comment by flipz357** [comment]
Bump. It seems like the PRs are ready to be merged.
